src/content/rooms/medium/library.coffee

```diff
@@ -43,7 +43,7 @@ Job.Library::next = add class LibraryDaily extends Page
     if Math.random() < 0.75  or g.events.LibraryDaily?[0] is g.day then return false
     l = [
       """|| bg="Library/Belly.jpg"
-        -- "Can I help you find anything, #{if @worker.gender is 'f' then "ma'am" else 'sir'}? M-my shirt? Yes, I-I suppose, if you wish..." """,
+        -- "Can I help you find anything, #{sir}? M-my shirt? Yes, I-I suppose, if you wish..." """,
```

Comment:
Haha, sorry if someone got pinged by my @ worker - didn't mean to make that reference a person.